packages/core/src/components/screens/AboutScreen.tsx

```diff
@@ -1,6 +1,63 @@
 import React from "react";
-import { Typography } from "@equinor/mad-components";
+import { Cell, EDSStyleSheet, Typography, useStyles } from "@equinor/mad-components";
+import { ScrollView, View } from "react-native";
+import { useMadConfig } from "../../store/mad-config";
 
 export const AboutScreen = () => {
-    return <Typography>Not implemented yet</Typography>;
+    const config = useMadConfig();
+    const styles = useStyles(themeStyles)
+    const environment = config.environment.charAt(0).toUpperCase() + config.environment.slice(1);
```

Comment:
Not strictly necessary but we have a `useEnvironment` hook you can use to select environment from madconfig, and `useAppVersion` for appVersion. You can also create a new hook `useAbout` for getting the about-config. Up to you if you prefer sticking to `useMadConfig` or want to use the other hooks, but in my opinion it leads to easier to understand code if we leave out everything we don't need.

Reply:
Thanks for the feedback 🙌 Switched to `useEnvironment` in [8773738](https://github.com/equinor/mad/pull/310/commits/87737389f1a61e1d48c5a6f543f1ebdf0fbbbc8a).

Reply:
Switched to `useAppVersion` in [d9bae07](https://github.com/equinor/mad/pull/310/commits/d9bae079a51f52a7a041ab3d0f4b9f8126f5db21).

Reply:
Switched to `useAbout` in [a8f98db](https://github.com/equinor/mad/pull/310/commits/a8f98db2e29a4742b4f173eb0e50361b699160a6).